tests/test_monitoring.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import json
+import subprocess
+import requests
+from dataclasses import dataclass
+from typing import List, Dict, Any
+
+
+@pytest.mark.order(8)
+def test_loki_labels():
+    loki_labels = get_loki_labels()
+
+    expected_labels = {"container_name", "compose_service", "compose_project"}
+    actual_labels = set(loki_labels.data)
+
+    assert expected_labels <= actual_labels
+
+
+@pytest.mark.order(9)
+def test_prometheus_labels():
+    prometheus_labels = get_prometheus_labels()
+
+    expected_labels = {"__name__", "instance", "job"}
+    actual_labels = set(prometheus_labels.data)
+
+    assert expected_labels <= actual_labels
+
+@dataclass
+class LabelsResponse:
+    data: List[str]
+
+    @classmethod
+    def from_dict(cls, response_dict: Dict[str, Any]) -> "LabelsResponse":
+        if "data" not in response_dict:
+            raise ValueError("Response missing required 'data' field")
+        return cls(data=response_dict["data"])
+
+
+def get_loki_labels() -> LabelsResponse:
+    response = requests.get("http://localhost:3100/loki/api/v1/labels", timeout=10)
+    response.raise_for_status()
+    return LabelsResponse.from_dict(response.json())
```

Comment:
Once you have a proper dataclass you can use `Labels.model_validate(response.json())` here